bot/exts/backend/error_handler.py

```diff
@@ -168,27 +173,28 @@ async def try_get_tag(self, ctx: Context) -> None:
         the context to prevent infinite recursion in the case of a CommandNotFound exception.
         """
         tags_get_command = self.bot.get_command("tags get")
```

Comment:
Change this to get the actual `tag` command as `self.bot.get_command` doesn't returns `app_commands`